README.md

```
@@ -1,3 +1,9 @@
 # StudioCMS Documentation Website
 
-https://docs.studiocms.dev
\ No newline at end of file
+https://docs.studiocms.dev
+
+### Translation progress
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)